src/Types.hs

```diff
@@ -56,10 +57,28 @@ data DiscoveredProject m = DiscoveredProject
   { projectType :: Text
   , projectPath :: Path Abs Dir
   , projectBuildTargets :: FoundTargets
-  , projectDependencyGraph :: FoundTargets -> m (Graphing Dependency)
+  , projectDependencyGraph :: FoundTargets -> m (Graphing Dependency, GraphBreadth)
   , projectLicenses :: m [LicenseResult]
   }
 
+-- | The exhaustiveness or completeness of the graph found during analysis.
+--   Complete - indicates that the dependencies in question are a full, transitive graph, requiring no additional analysis
+--      Ex -> yarn.lock, Podfile.lock, Gemfile.lock
+--   Partial - indicates that the dependencies in question do *NOT* represent the fully resolved graph, i.e. because of a limitation
+--             in the package manager in use, it wasn't possible for us to obtain a full list of deps
+--      Ex -> stand-alone `package.json`, stand-alone Podfile
```

Comment:
[nit]: It might be worth mentioning that the ramification of a `Partial` graph is that it is re-run recursively.